modules/yoastSeo/yoastSeo.php

```diff
@@ -0,0 +1,149 @@
+<?php
+
+use PublishPress\Checklists\Core\Factory;
+use PublishPress\Checklists\Core\Legacy\LegacyPlugin;
+use PublishPress\Checklists\Core\Legacy\Module;
+use PublishPress\Checklists\Yoastseo\Requirement\ReadabilityAnalysis;
+use PublishPress\Checklists\Yoastseo\Requirement\SeoAnalysis;
+
+/**
+ * @package     PublishPress\Checklists
+ * @author      PublishPress <help@publishpress.com>
+ * @copyright   copyright (C) 2019 PublishPress. All rights reserved.
+ * @license     GPLv2 or later
+ * @since       1.0.0
+ */
+
+/**
+ * Class PPCH_Yoastseo
+ *
+ * @todo Refactor this module and all the modules system to use DI.
+ */
+class PPCH_Yoastseo extends Module
```

Comment:
The class name and file name are not consistent. The file is called "yoastSeo", and the class "PPCH_Yoastseo" (different case pattern). I think the class name is good, but the module, and file name should be "yoastseo", all lower case. Please, rename the module.

Reply:
The reason for "PPCH_" in the class name is because all modules takes "PPCH_" prefixes in the class name even though it's not in any of the file name and without the "PPCH_" prefix, the class won't load as it's part of the public mode of loading class. You can check other modules class name.

Reply:
@olatechpro I wasn't clear, sorry. I was not talking about that prefix. I know it is required. I was talking about the file name and module name. Instead of `yoastSeo`, it should be `yoastseo`. So the path would be `modules/yoastseo/yoastseo` and not `modules/yoastSeo/yoastSeo`. The class name is correct: PPCH_Yoastseo. That is why I asked to rename the module, not the class 👍